docs/Developer Resources/Building ZFS.rst

```diff
@@ -4,25 +4,24 @@ Building ZFS
 GitHub Repositories
 ~~~~~~~~~~~~~~~~~~~
 
-The official source for ZFS on Linux is maintained at GitHub by the
-`zfsonlinux <https://github.com/zfsonlinux/>`__ organization. The
-project consists of two primary git repositories named
-`spl <https://github.com/zfsonlinux/spl>`__ and
-`zfs <https://github.com/zfsonlinux/zfs>`__, both are required to build
-ZFS on Linux.
-
-**NOTE:** The SPL was merged in to the
-`zfs <https://github.com/zfsonlinux/zfs>`__ repository, the last major
-release with a separate SPL is ``0.7``.
-
--  **SPL**: The SPL is thin shim layer which is responsible for
-   implementing the fundamental interfaces required by OpenZFS. It's
-   this layer which allows OpenZFS to be used across multiple platforms.
+The official source for OpenZFS is maintained at GitHub by the
+`openzfs <https://github.com/openzfs/>`__ organization. The primary
+git repository for the project is the `zfs
+<https://github.com/openzfs/zfs>`__ repository.
+
+There are two main components in this repository:
 
--  **ZFS**: The ZFS repository contains a copy of the upstream OpenZFS
-   code which has been adapted and extended for Linux. The vast majority
-   of the core OpenZFS code is self-contained and can be used without
-   modification.
+- **ZFS**: The ZFS repository contains a copy of the upstream OpenZFS
```

Comment:
In terms of eliminating the "copy of upstream", maybe something like this:

**ZFS**: The ZFS repository contains the OpenZFS code which supports FreeBSD and Linux. The vast majority of the OpenZFS code is OS-independent.

Reply:
@rlaager I agree "copy of upstream" should be gone. However:

"The vast majority of the OpenZFS code is OS-independent."

 What do you want to say here?
- The original meaning was the fact it was "self contained" and usable without modification.
- Your version doesn't seem to have the same meaning, it's rather vague.
- also "The ZFS repository contains the OpenZFS code which supports FreeBSD and Linux", is rather vague too... It could just as well be read "contains compatibility code" or "Contains code that supports FreeBSD and Linux" (without being clear it's the main repo for the project)

How about something like, to keep the original meaning in tact:
```suggestion
- **ZFS**: The ZFS repository contains the main OpenZFS code with support for FreeBSD and Linux. The vast majority of the OpenZFS code is OS-independent and can be used without modification.
```

Reply:
In terms of "self contained" vs "OS-independent", I actually have the same question: I have no idea what the original intention was.

If the audience of this is a user of ZFS, rather than someone porting it to a new platform, I'm not sure that we need the second sentence anyway.

Reply:
@rlaager You are right, I was getting confused by that second scentence., reading both versions this tells all we need:

```suggestion
- **ZFS**: The ZFS repository contains the OpenZFS code which supports FreeBSD and Linux.
```
_(or my version of the same scentence, thats more of a style choice)_

That being said:
Shouldn't there be a comma after code?
`The ZFS repository contains the OpenZFS code, which supports FreeBSD and Linux.`
vs
`The ZFS repository contains the OpenZFS code which supports FreeBSD and Linux.`

afaik one of those is:
"the code to supports something" and the other "the code having/including support for something"